source/behavior/entities/format/components/projectile.json

```diff
@@ -585,9 +608,18 @@
         },
         "teleport_owner": {
           "title": "Teleport Owner",
-          "type": "boolean",
-          "default": false,
-          "description": "Determines if the owner is transported on hit."
+          "description": "Determines if the owner is transported on hit.",
+          "anyOf": [
+            {
+              "type": "boolean",
+              "default": false
+            },
+            {
+              "type": "object", 
+              "additionalProperties": false,
+              "properties": {}
+            }
+          ]
         },
```

Comment:
The docs say it should be a boolean value, while vanilla files do things sometimes in their own way, Which might also be wrong, And not even sure those values work. But I would rather enforce it here as a boolean value as it's what is documented. In case they fix it, and we support something they will fix in the future

Reply:
I'll test it out in game and see which of those work, if Boolean works I'll leave it as that.

Reply:
Both values work, so I'll leave it as boolean